### ISSUE TITLE
cifar10 no response

### ISSUE THREAD

**Issue by considerxzh** [jhfjhfj1/autokeras#587]
### Bug Description

Using TensorFlow backend.
Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.
Saving Directory: /tmp/autokeras_E8R5E4
Preprocessing the images.
Preprocessing finished.

Initializing search.
Initialization finished.

+----------------------------------------------+
|               Training model 0               |
+----------------------------------------------+
                                                                                                    
No loss decrease after 5 epochs.

Saving model.
+--------------------------------------------------------------------------+
|        Model ID        |          Loss          |      Metric Value      |
+--------------------------------------------------------------------------+
|           0            |   3.9836374521255493   |   0.6620000000000001   |
+--------------------------------------------------------------------------+

+----------------------------------------------+
|               Training model 1               |
+----------------------------------------------+
                                                                                                    
No loss decrease after 5 epochs.

Saving model.
+--------------------------------------------------------------------------+
|        Model ID        |          Loss          |      Metric Value      |
+--------------------------------------------------------------------------+
|           1            |   2.9820535719394683   |         0.8036         |
+--------------------------------------------------------------------------+

+----------------------------------------------+
|               Training model 2               |
+----------------------------------------------+
                                                                 
No loss decrease after 5 epochs.

Saving model.
+--------------------------------------------------------------------------+
|        Model ID        |          Loss          |      Metric Value      |
+--------------------------------------------------------------------------+
|           2            |   2.642572206258774    |   0.8139999999999998   |
+--------------------------------------------------------------------------+

+----------------------------------------------+
|               Training model 3               |
+----------------------------------------------+
                                                                                                    
No loss decrease after 5 epochs.

### Reproducing Steps
 here is my code:
`from keras.datasets import cifar10
from autokeras import ImageClassifier
from autokeras.constant import Constant
import time

if __name__ == '__main__':
    (x_train, y_train), (x_test, y_test) = cifar10.load_data()
    a=time.time()
    clf = ImageClassifier(verbose=True, augment=False)
    clf.fit(x_train, y_train, time_limit=12* 60 * 60)
    clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
    y = clf.evaluate(x_test, y_test)
    b=time.time()
    print('time costs {}'.format(b-a))
    print(y * 100)`

### Expected Behavior
the log file have added information of new model id 3, 
it should save model and keep training， but gets stuck here

### Setup Details
Include the details about the versions of:
 - OS type and version:Ubuntu 16.0.4
 - Python: 3.6
 - autokeras: 0.3.7
 - scikit-learn:0.20.2
 - numpy:1.16.1
 - keras:2.2.4
 - scipy:1.2.0
 - tensorflow:1.3.0
 - pytorch:1.0.1.post2

**Comment by considerxzh** [comment]
It go on training after 8 or 9 hour, and finished test by time out. Does searching time cost this too long?
